rsplan/unit_tests.py

```diff
@@ -88,7 +88,7 @@ def test_straight_path(
     )
 
     for wp1, wp2 in zip(nav_path.waypoints()[:-1], nav_path.waypoints()[1:]):
-        assert wp1 != wp2
+        assert not wp1.is_close(wp2)
```

Comment:
tolerance?

Reply:
Since the waypoint class has the properties `curvature` and  `is_runway` which are describing the segment the waypoint is coming from. I think we are not looking for true duplicates where hash(wp1) == hash(wp2) but a pose true duplicate:
```
wp1.pose_2d_tuple != wp2.pose_2d_tuple
```
`Waypoint.is_close()` checks this but with a `1e-5` tolerance. I updated the code to use the pose check instead.